lib/razorpay/addon.rb

```diff
@@ -0,0 +1,19 @@
+require 'razorpay/request'
+require 'razorpay/entity'
+
+module Razorpay
+  # Addon API allows you to fetch and delete
+  # subscription-addons with Razorpay
+  class Addon < Entity
+    def self.request
+      Razorpay::Request.new('addons')
+    end
+
+    def self.fetch(id)
+      request.fetch id
+    end
+
+    ## delete response is blank Array
+    ## getting consensus on how to better handle it
```

Comment:
In the interests of getting this merged faster, I'd be okay without merging without a delete method for now. Please remove the comments if you agree.

Reply:
Comments removed :white_check_mark: